Example/Core/Tests/GTMHTTPServer.h

```diff
@@ -0,0 +1,145 @@
+/* Copyright (c) 2010 Google Inc.
```

Comment:
Is this really from 2010? Remove the "(c)" to be consistent with Firebase copyrights and pass travis